app/views/posts/index.html.erb

```diff
@@ -1 +1,33 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<section class="user-container">
+  <card class="user-item">
+    <img src='<%= @user.photo %>'></img>
+    <div class="user-data">
+    <h2><%= @user.name %></h2><br>
+    <h4 class="custom-card-t">Number of posts: <%= @user.posts_counter %> </h4>
+    </div>
+  </card>
+  <div class="posts-container">
+    <% @posts.each do |post| %>
+      <div class="post-comment-box">
+        <card class="post-item">
+          <div class="post-data">
+            <h2>Post <%= post.id %></h2>
+            <p> <%= post.text %> </p>
+            <div class="counter-data">
+              <h5 class="custom-card-p">Comments: <%= post.comments_counter %> </h5>
+              <h5 class="custom-card-p">Likes: <%= post.likes_counter %> </h5>
+            </div>
+          </div>
+        </card>
+        <div class="comments-box">
+          <% if post.comments.empty? %>
+            <p> There are no comments for this post </p>
+          <% end %>
+          <% post.comments.each do |comment| %>
+            <p> <%= comment.author.name %>: <%= comment.text %> </p>
+          <% end %>
+        </div>
+      </div>
+    <% end %>
+  </div>
+</section>
```

Comment:
![image](https://github.com/NelsonAraujo93/RoR-Blog/assets/93039359/0b981a47-0486-438c-9d8b-c3b353b9aa93)

- [ ] It would be nice if you make a Pagination button at the bottom which you can see in the design 👍 to make your project follow the requirements 😊